pom.xml

```diff
@@ -5,12 +5,12 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.7.7</version>
+        <version>3.2.0</version>
     </parent>
 
     <groupId>com.github.yoep</groupId>
     <artifactId>spring-boot-starter-javafx</artifactId>
-    <version>1.0.13-SNAPSHOT</version>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
I would actually make this a new major version (2.0.0) as we're bumping Spring Boot to a new major version as well as the java version which might break this plugin for some developers.

Reply:
You're totally right and I was in doubt about this one myself...
Changed it!

I ran the unit tests, or other tests needed?

Would you mind getting in contact via mail? I would really love to know more about this library, how you (and others) use it, your motivation to make it, etc. Would be an interesting topic for a blog post or live stream...